README.md

```diff
@@ -51,7 +51,12 @@ steps:
 
 ## Configuration
 
+### `api-secret-env` (Optional, string)
+
+The environment variable that the Wiz API Secret is stored in. Defaults to using `WIZ_API_SECRET`.
```

Comment:
I think we should include a link to our docs on secrets for folks to have a read through (if they haven't already), since we are asking them to bring their own secrets with this pattern.

```suggestion
The environment variable that the Wiz API Secret is stored in. Defaults to using `WIZ_API_SECRET`. Refer to the [documentation](https://buildkite.com/docs/pipelines/secrets#using-a-secrets-storage-service) for more information about managing secrets on your Buildkite agents.
```